apps/api/v2/src/modules/users/inputs/create-managed-user.input.ts

```diff
@@ -1,8 +1,8 @@
-import { IsTimeFormat } from "@/modules/users/inputs/validators/is-time-format";
-import { IsWeekStart } from "@/modules/users/inputs/validators/is-week-start";
 import { ApiProperty } from "@nestjs/swagger";
-import { IsNumber, IsOptional, IsTimeZone, IsString, Validate } from "class-validator";
+import { IsOptional, IsTimeZone, IsString } from "class-validator";
 
+export type WeekDay = "Monday" | "Tuesday" | "Wednesday" | "Thursday" | "Friday" | "Saturday" | "Sunday";
```

Comment:
if can we put the list of days in a Const, we can use 

type weekDay = (type WEEKDAYS)[number]